src/Client/scripts/App.tsx

```diff
@@ -43,6 +46,13 @@ const App: React.FC<{}> = () => {
       'Projects': projects
    };
 
+  
+   const [resumeOpen, setResumeOpen] = useState(false);
```

Comment:
```const [resumeOpen, setResumeOpen] = React.useState<boolean>(null);``` for type

Reply:
And move this code to line 34 (to be defined at the place where similar thing defined)

Reply:
fixed!